app/src/main/java/com/example/stromtracker/ui/haushalt/HaushaltFragment.kt

```diff
@@ -51,16 +50,17 @@ class HaushaltFragment : Fragment() {
                     datain.addAll(haushalte)
                     //Die Haushaltliste in Main Activity erneuern.
                     mainact.setOldHaushaltList(datain)
+                    //Standardräume erstellern, sobald ein neuer Haushalt erzeugt wird.
+                    if (datain.size > datatemp.size && !isinit&&datatemp.size>0) {
+                        initRaeume(datain[datain.size - 1].getHaushaltID())
+                    }
+                    //Neue Räume erzeugen, anchdem ein Brnad neuer Erzeugt wurde.
                     if (isinit && datain.size > 0) { //Sobald neues Haushalt erstellt wurde, sollen ein paar Standardräume erzeugt werden.
                         // Hier nach leer initialisierung und sobald das ganze in die Datenbank gekommen ist,deswegen wird hier gewartet, bis datain bei einem element ist,
                         //bevor es in die if Schleife kommt
                         initRaeume(datain[datain.size - 1].getHaushaltID())
                         isinit = false
```

Comment:
Wir erzeugen ja in der Datenbank schon einen Raum für den ersten Haushalt, wird der hier doppelt erzeugt?

Reply:
Nein das verhindere ich durch das init.
Für den wird nur der eine Raum erzeugt bis jetzt und zwar Sonstiges. 
Hab ich auch getestet und es passiert nicht, dass Raeume doppelt erzeugt werden.